shared/validation/delivery-area-validator.ts

```diff
@@ -1,14 +1,8 @@
-export function isPostcodeValid(postcode: string): boolean {
+export function isPostcodeStartWithValidValue(postcode: string): boolean {
```

Comment:
This should probably be named something like `isPostcodeWithinDeliveryArea` because the function doesn't check the validity of a postcode, it just checks if it's inside the delivery areas